#! /bin/sh /usr/share/dpatch/dpatch-run
## 60-dont-open-userprefs.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Dont open user preferences for spamassassin

@DPATCH@
diff -urNad trunk/spampd /tmp/dpep.JxwWR4/trunk/spampd
--- trunk/spampd	2005-12-22 23:32:09.000000000 +0100
+++ /tmp/dpep.JxwWR4/trunk/spampd	2005-12-22 23:32:48.375183506 +0100
@@ -891,7 +891,7 @@
    $assassin->set_persistent_address_list_factory ($addrlistfactory);
 };
 
-$assassin->compile_now();
+$assassin->compile_now(0);
 
 # thanks to Kurt Andersen for the 'uname -s' fix
 if ( !$options{logsock} ) {
